server/api/rounds.py

```diff
@@ -565,23 +572,35 @@ def draw_sample_for_contest(
         # Compute the total number of ballot samples in all rounds leading up to
         # this one. Note that this corresponds to the number of SampledBallotDraws,
         # not SampledBallots.
-        num_previously_sampled = SampledBallotDraw.query.filter_by(
-            contest_id=contest.id
-        ).count()
+        num_previously_sampled = (
+            SampledBallotDraw.query.filter_by(contest_id=contest.id)
+            .join(SampledBallot)
+            .join(Batch)
+            .filter_by(has_cvrs=filter_has_cvrs)
```

Comment:
just checking: you explicitly want this to select for `has_cvrs = null` in the case of ballot comparison and batch comparison?

Reply:
yes, will add a comment